### ISSUE TITLE
Suggestion: Better handling of malformed settings file

### ISSUE THREAD

**Issue by hk21702** [RimSort/RimSort#501]
Add a dialogue specific to when the settings file is malformed and cannot be properly loaded with the following options:

> Reset Settings
> Open settings.json file
> Quit

This dialogue should appear as a last resort option if something goes wrong  (i.e., a critical failure) when trying to parse the settings file. If the issue is something like an improper path that can be fixed by just resetting those specific entries, then this dialogue should not open unless the proper mitigation method has also failed.

**Comment by oceancabbage** [comment]
Implemented related solution in https://github.com/RimSort/RimSort/pull/532
